grails-app/i18n/messages.properties

```
@@ -1073,6 +1074,29 @@ fulfillmentItem.available.label=Available
 fulfillmentItem.packed.label=Packed
 fulfillmentItem.picked.label=Picked
 fulfillmentItem.requested.label=Requested
+
+# Messages related to outbound import
+fulfillmentRequest.origin.invalid=Origin can't be the same as destination
+fulfillmentRequest.destination.invalid=Destination can't be the same as origin
```

```suggestion
fulfillmentRequest.origin.sameAsDestination=Origin can't be the same as destination
fulfillmentRequest.destination.sameAsOrigin=Destination can't be the same as origin
```
Because "invalid" is very general but the error is a very specific error scenario.